src/app/src/pages/eval/components/ResultsView.tsx

```diff
@@ -864,6 +901,27 @@ export default function ResultsView({
         onShare={handleShare}
       />
       <SettingsModal open={viewSettingsModalOpen} onClose={() => setViewSettingsModalOpen(false)} />
+      <ConfirmEvalNameDialog
+        open={editNameDialogOpen}
+        onClose={() => setEditNameDialogOpen(false)}
+        title="Edit Eval Name"
+        label="Description"
+        currentName={config?.description || ''}
+        actionButtonText="Save"
+        onConfirm={handleSaveEvalName}
+      />
+      <ConfirmEvalNameDialog
+        open={copyDialogOpen}
+        onClose={() => setCopyDialogOpen(false)}
+        title="Copy Evaluation"
+        label="Description"
+        currentName={`${config?.description || 'Evaluation'} (Copy)`}
```

Comment:
The inclusion of "(Copy)" is confusing here since this is current, _original_ name.